test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
```

Comment:
We don't use test folder, so you add `test` into `.gitignore` and run `git rm --cached -r test`